velox/benchmarks/basic/FormatDateTimeBenchmark.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/Benchmark.h>
+#include <folly/init/Init.h>
+#include "velox/benchmarks/ExpressionBenchmarkBuilder.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+
+using namespace facebook;
+
+using namespace facebook::velox;
+
+int main(int argc, char** argv) {
+  folly::Init init(&argc, &argv);
+  functions::prestosql::registerDateTimeFunctions("");
+
+  ExpressionBenchmarkBuilder benchmarkBuilder;
+  const vector_size_t vectorSize = 1000;
+  auto vectorMaker = benchmarkBuilder.vectorMaker();
+  auto timestampInput =
+      vectorMaker.flatVector<Timestamp>(vectorSize, [&](auto j) {
+        return Timestamp(1695859694 + j / 1000, j % 1000 * 1'000'000);
+      });
+
+  benchmarkBuilder
+      .addBenchmarkSet("", vectorMaker.rowVector({"input"}, {timestampInput}))
+      .addExpression(
+          "format_datetime",
+          "format_datetime(input, 'yyyy-MM-dd HH:mm:ss.SSS')")
+      .withIterations(1000)
```

Comment:
Is this needed? CC: @laithsakka

Reply:
1000 iterations may be better.

Reply:
Curious, how do you decide whether 1000 is better than default (100?). Would you clarify? Perhaps, add a comment in the code to help future readers.

Reply:
@mbasmanova , I just found the default one is 1000. Maybe, we can just remove the iteration setting. Right?
https://github.com/facebookincubator/velox/blob/7822fe154b9d24d4e50bbc41f0c4ca5d39e2db03/velox/benchmarks/ExpressionBenchmarkBuilder.h#L91

Reply:
Yes, let's remove. Thanks.